src/main/java/org/eolang/jeo/representation/xmir/XmlNode.java

```diff
@@ -224,13 +233,9 @@ XmlBytecodeEntry toEntry() {
      */
     private Optional<XmlNode> optchild(final String name) {
         Optional<XmlNode> result = Optional.empty();
-        final NodeList children = this.node.getChildNodes();
-        for (int index = 0; index < children.getLength(); ++index) {
-            final Node current = children.item(index);
-            if (current.getNodeName().equals(name)) {
-                result = Optional.of(new XmlNode(current));
-                break;
-            }
+        final List<XML> nodes = new XMLDocument(this.node).nodes(name);
```

Comment:
@h1alexbel The code is much simpler, indeed! What is a bit strange, that we parse `xml` node in the constructor:
```java
public XmlNode(final XML xml) {
  this(xml.node());
}
```

and then create it again:

```java
final List<XML> nodes = new XMLDocument(this.node).nodes(name);
```

I understand, why you do this, but maybe we can replace `private final Node node;` field with `private final XML node;` ? What do you think?If you’re okay with it, it’s better to add one more puzzle.

Reply:
@volodya-lombrozo I thought about the replacement of `Node` with `XML`. I will replace it in this pull